.gitpod.Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Note: gitpod/workspace-base image references older version of CMake, it's necessary to install newer one
+FROM  gitpod/workspace-base
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+ARG CONTAINER_USER=gitpod
+ARG CONTAINER_GROUP=gitpod
+ARG NIGHTLY_VERSION=nightly-2022-03-10
+ARG ESP_IDF_VERSION=release/v4.4
```

Comment:
With #80 merged, could you update the env var to `v4.4.1` (as in [`Dockerfile`](https://github.com/ferrous-systems/espressif-trainings/blob/main/.devcontainer/Dockerfile#L9))?

Reply:
Updated and verified that all the projects are building fine!